core/src/main/java/io/questdb/cairo/TableUtils.java

```diff
@@ -1024,6 +1024,9 @@ public static void setNull(int columnType, long addr, long count) {
                 assert Long128Constant.NULL_HI == Long128Constant.NULL_LO;
                 Vect.setMemoryLong(addr, Long128Constant.NULL_HI, count * 2);
```

Comment:
calling setMem is excessive here

Reply:
why do you think so? it's setting null across potentially many rows. 
it's the same for existing types.